lib/marked.js

```diff
@@ -554,9 +554,48 @@ inline.normal = merge({}, inline);
 inline.pedantic = merge({}, inline.normal, {
   strong: /^__(?=\S)([\s\S]*?\S)__(?!_)|^\*\*(?=\S)([\s\S]*?\S)\*\*(?!\*)/,
   em: /^_(?=\S)([\s\S]*?\S)_(?!_)|^\*(?=\S)([\s\S]*?\S)\*(?!\*)/,
-  link: edit(/^!?\[(label)\]\(\s*<?([\s\S]*?)>?(?:\s+(['"][\s\S]*?['"]))?\s*\)/)
-    .replace('label', inline._label)
-    .getRegex(),
+  link: {
+    exec: function (s) {
+      // [TEXT](DESTINATION)
+      var generalLinkRe = edit(/^!?\[(label)\]\((.*?)\)/)
+        .replace('label', inline._label)
+        .getRegex();
+
+      function unwrapCarats (str) {
```

Comment:
Carets?? Carat and carrot are different. :)

Angle brackets might be most appropriate if I'm reading the regex correctly.

Reply:
Haha I was wondering if I was spelling that right. I'll switch to AngleBrackets anyway.